x-pack/plugins/security_solution/cypress/tasks/lists.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  VALUE_LISTS_MODAL_ACTIVATOR,
+  VALUE_LIST_FILE_PICKER,
+  VALUE_LIST_FILE_UPLOAD_BUTTON,
+} from '../screens/lists';
+
+export const waitForListsIndexToBeCreated = () => {
+  cy.request({ url: '/api/lists/index', retryOnStatusCodeFailure: true }).then((response) => {
+    if (response.status !== 200) {
+      cy.wait(7500);
+    }
+  });
+};
+
+export const waitForValueListsModalToBeLoaded = () => {
+  cy.get(VALUE_LISTS_MODAL_ACTIVATOR).should('exist');
+  cy.get(VALUE_LISTS_MODAL_ACTIVATOR).should('not.be.disabled');
+};
+
+export const openValueListsModal = () => {
+  cy.get(VALUE_LISTS_MODAL_ACTIVATOR).click();
+};
+
+export const selectValueListsFile = () => {
```

Comment:
In this method I'll pass the name of the file to be attached instead of hardcoding it ;)